service/cloudconfig/cloud_config.go

```diff
@@ -1,43 +1,267 @@
 package cloudconfig
 
 import (
+	"github.com/giantswarm/certificatetpr"
 	k8scloudconfig "github.com/giantswarm/k8scloudconfig"
-)
+	"github.com/giantswarm/microerror"
 
-type MasterExtension struct{}
+	"github.com/giantswarm/azure-operator/service/key"
+)
 
-// Files allows files to be injected into the master cloudconfig. It is
-// currently empty because the certificates will be stored in Key Vault.
+// Files allows files to be injected into the master cloudconfig.
 func (me *MasterExtension) Files() ([]k8scloudconfig.FileAsset, error) {
-	return nil, nil
+	var newFiles []k8scloudconfig.FileAsset
```

Comment:
A single file is injected which is the script for getting the secrets. More files can be added later if needed.